lib/src/main/java/com/diffplug/spotless/npm/NpmFormatterStepStateBase.java

```diff
@@ -109,12 +114,13 @@ protected ServerProcessInfo npmRunServer() throws ServerStartException, IOExcept
 			assertNodeServerDirReady();
 			LongRunningProcess server = null;
 			try {
-				// The npm process will output the randomly selected port of the http server process to 'server.port' file
+				final UUID nodeServerInstanceId = UUID.randomUUID();
+				// The npm process will output the randomly selected port of the http server process to 'server-<id>.port' file
```

Comment:
Seems like we don't need the UUID because we have the ExclusiveFolderAccess?

Reply:
I'm using the ExclusiveFolderAccess to make sure that `npm install` is not run concurrently. Launching the server in parallel, however, is now explicitly supported and thus we use the `UUID` to give each java thread its corresponding running npm server (differentiated via the UUID passed on launch). Hope that clears things up.